src/pipecat/processors/filters/stt_mute.py

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (c) 2024, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+from dataclasses import dataclass
+from enum import Enum
+from typing import Callable, Optional
+
+from loguru import logger
+
+from pipecat.frames.frames import (
+    BotStartedSpeakingFrame,
+    BotStoppedSpeakingFrame,
+    Frame,
+    StartInterruptionFrame,
+    StopInterruptionFrame,
+    STTMuteFrame,
+)
+from pipecat.processors.frame_processor import FrameDirection, FrameProcessor
+from pipecat.services.ai_services import STTService
+
+
+class STTMuteStrategy(Enum):
+    NEVER = "never"  # Never mute
```

Comment:
Do we need a `NEVER` strategy?

Reply:
Possibly not. If you want variable muting, you can use CUSTOM to hook into your own function via callback. Maybe we remove it?

Reply:
Sorry, we actually don't need it and we should force the user to specify the strategy, that is, not make it optional.

Reply:
Which means we don't pass `STTMuteConfig()`

Reply:
Yup, I agree. I'll make this change.